packages/algolia-experiences/src/setup-instantsearch.ts

```diff
@@ -39,3 +44,35 @@ export function setupInstantSearch() {
     error((err as Error).message);
   }
 }
+
+function registerComponents(search: InstantSearch, settings: Settings) {
+  class AlgoliaExperience extends HTMLElement {
+    static observedAttributes = ['experience-id'];
+
+    widgets: IndexWidget[] = [];
+
+    connectedCallback() {
+      const experienceId = this.getAttribute('experience-id');
```

Comment:
wonder if we should use the same data-attribute so it's easier to switch between both modes?

Reply:
The problem with this is that when loading initially it would attempt to render twice via the `querySelector` and within the lifecycle itself, as `<algolia-experiences data-experience-id="..." />` would match.
We would have to add more checks to ensure it doesn't happen.

Not sure there would be a scenario where you would wanna switch between both modes, except if we want to do custom elements only and ask people to change their code

Reply:
I'm ok with custom experiences only, nobody has implemented this yet, maybe the querySelector code goes away all together then?